ndk/src/trace.rs

```diff
@@ -0,0 +1,94 @@
+//! Bindings for the NDK tracing API.
+//!
+//! See also [the NDK docs](https://developer.android.com/ndk/reference/group/tracing)
+#![cfg(feature = "trace")]
+use std::ffi::{CString, NulError};
+
+pub fn is_trace_enabled() -> bool {
+    unsafe { ffi::ATrace_isEnabled() }
+}
+
+#[derive(Debug)]
+pub struct Section {
+    _private: *mut (),
```

Comment:
minor: a comment should be added why this field is required. I would also suggest to rather use `PhantomMarker` if it's soley to enforce !Send+!Sync

Reply:
Indeed, didn't have to use a pointer here. Thanks for the suggestion!